recipes/pyads/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pyads" %}
+{% set version = "3.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/stlehmann/{{ name }}/releases/download/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ce00b9875a98f1d98448afb6be81978d5c37736de64ba9b9a432267a455fcbb7
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<36]
```

Comment:
Can this be noarch python?

Reply:
There is a C++ library that needs to be built on Linux / OS X patforms but not on Windows. So I assumed noarch python is not an option...?

Reply:
Shall I still change to noarch python?

Reply:
@scopatz What can I do to make progress with this PR? I still don't think noarch would be the right choice here because there is a C library that needs to be built on Linux. Not so on Windows because there it relies on 3rd party software. I appreciate any help to make progress with this PR :)

Reply:
Can't be noarch due to cxx portion